policyengine_us/variables/gov/states/ct/tax/income/ct_amt.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class ct_amt(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Connecticut alternative minimum tax"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.CT
+    reference = (
+        "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/2022-CT-1040-Instructions_1222.pdf#page=2"  # line 9
+        "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/CT-1040_1222.pdf#page=1"  # line 9
+        "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/CT-6251_1222.pdf#page=1"  # line 23
+    )
+
+    def formula(tax_unit, period, parameters):
+        # assign amt parameters
+        p = parameters(period).gov.states.ct.tax.income
+        amt = p.alternative_minimum_tax.rate
+
+        ct_income_tax_after_personal_credits = tax_unit(
+            "ct_income_tax_after_personal_credits", period
+        )
```

Comment:
do we represent that: ?
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/4e8c1578-e187-46b8-970e-19565fee2c29)

Reply:
Line 2 : https://portal.ct.gov/-/media/DRS/Forms/2022/Income/CT-6251_1222.pdf#page=1

Reply:
same for the subtractions

Reply:
could you comment out which steps of this form are form the federal form and which are computed here 
(add lines to computation steps)

https://portal.ct.gov/-/media/DRS/Forms/2022/Income/CT-6251_1222.pdf#page=1